src/main/resources/application.yml

```diff
@@ -9,10 +12,14 @@ security:
 spring:
   boot:
     admin:
-      url: http://spring-boot-admin:8080
-      auth:
-        username: admin
-        password: password
+      url: http://localhost:${server.port}
+      username: ${security.user.name}

```

Comment:
Shouldn't this be (spring.boot.admin.)auth.username
And .password below as well?

Reply:
I am not familiar with "${security.user.*}". Is this referencing another variable that is to be provided externally?

Reply:
Both values are defined in the file a couple lines above, and define how the whole application is secured. The same values are required when the application wants to register at itself.

Before, it was working more or less by coincidence, because both values were the same. Now, you can run the app with runtime-configuration for `security.user.name` and `security.user.password` and self-registration will work without specifying the same values again.

Also, the namespace was wrong before; the `auth` part is not used, at least in the current version.

If you start the app now with `./gradlew bootRun` it will self-register itself right away.

Reply:
Ok, thanks for the explanation. I'll merge this then.